packages/synapse-react-client/src/style/base/_core.scss

```diff
@@ -721,15 +720,13 @@ main,
       @include SRC_Mixins.ignoreLink;
     }
   }
-  *,
-  .bootstrap-4-backport {
-    a.ignoreLink:not(.btn, .MuiLink-root, .MuiButton-root) {
+  // TODO: Remove * ?
+  * a.ignoreLink:not(.btn, .MuiLink-root, .MuiButton-root) {
+    @include SRC_Mixins.ignoreLink;
+    &:visited,
+    &:active,
+    &:link {
       @include SRC_Mixins.ignoreLink;
-      &:visited,
-      &:active,
-      &:link {
-        @include SRC_Mixins.ignoreLink;
-      }
     }
   }
```

Comment:
This selector touches a lot, and isn't bootstrap-specific, so I left it--even though I want to remove it.